code/modules/research/designs/autolathe_designs.dm

```diff
@@ -420,6 +420,14 @@
 	build_path = /obj/item/reagent_containers/glass/beaker/large
 	category = list("initial", "Medical")
 
+/datum/design/solids
+	name = "Solid Beaker"
+	id = "solids"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 500)
+	build_path = /obj/item/reagent_containers/glass/beaker/solids
+	category = list("initial", "Medical")
+
```

Comment:
Moving solid beakers which are only explicitly for making grenades and are objectively worse than normal beakers.
No.
If you want to do something like this, just add glass bottles (30u) as an available option for grenades.

Reply:
:+1: done